web/dashboard/src/styles/page-globals.css

```
@@ -238,19 +238,43 @@
-    background: var(--bg-surface);
-    box-shadow: 0 8px 24px color-mix(in srgb, var(--bg) 70%, transparent);
+  position: fixed;
+  top: 16px;
+  left: var(--sidebar-width);
+  right: 0;
+  margin: 0 auto;
+  width: fit-content;
+  z-index: 110;
+  pointer-events: none;
+  --loading-state-min-height: 0;
+  padding: 10px 14px;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)